mgr/ConsDefs.pm

```diff
@@ -1182,13 +1182,18 @@
     } else {
 	print "Could not find xml libs\n" if (! $param::quiet);
     }
- #Vc check SSE support
- my $cmd = "touch temp_gccflags.c; $CXX -E -dM -o - temp_gccflags.c | grep -q SSE";
- my $VcCPPFLAGS = " -DVC_IMPL=SSE";
- if ($STAR_HOST_SYS =~ 'gcc432$' || system($cmd)) {# No SSE
-   $VcCPPFLAGS = " -DVC_IMPL=Scalar";
-   if (-e "temp_gccflags.c") {`rm temp_gccflags.c`;}
- }
+
+    my $VcCPPFLAGS = " -I" . $VC_DIR . "/include" . " -Wabi -fabi-version=0";
+    my $VcLIB = $VC_DIR . "/lib/libVc.a";
```

Comment:
Why don't you define $LIBPATH for Vc? It is defined for other packages.
By hardcoding the library name as `libVc.a` you make the build configuration less portable.

Reply:
I can be defined. We both know the Vc only has this static library, it may not add much flexibility, but yes, it can be done.